script.js

```diff
@@ -1,4 +1,150 @@
+// Roll dice and put them in order and make highest number possible
+// User will roll dices - include dice function
+// Arrange numbers - string
+// User win if their number  is bigger
+// user lose if their number is smaller
+
+// Need to add while to loop entire function for player 1,2,3.
+
+// Variables
+var myOutputValue = "";
+var currentGameMode = "player 1";
+
+// Dice Roll
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+//Diceroll Input (1)
+var player1Dice1 = diceRoll();
+console.log(`Player 1 Dice 1 rolled ${player1Dice1}`);
+
+var player1Dice2 = diceRoll();
+console.log(`Player 1 Dice 2 rolled ${player1Dice2}`);
+
+//Diceroll Input (2)
+var player2Dice1 = diceRoll();
+console.log(`Player 2 Dice 1 rolled ${player2Dice1}`);
+
+var player2Dice2 = diceRoll();
+console.log(`Player 2 Dice 2 rolled ${player2Dice2}`);
```

Comment:
Good that you are storing your values in well named variables! Maybe you can explore storing them in arrays like [[P1D1,P1D2] , [P2D1,P2D2]] or objects  playerOneProfile = { dice : [  D1, D2], score : 42, etc}